Tests/SwiftLintFrameworkTests/ConfigurationTests+MultipleConfigs.swift

```diff
@@ -301,6 +301,113 @@ extension ConfigurationTests {
+            TestCase(optedInInParent: true, disabledInParent: true, optedInInChild: false, disabledInChild: false, isEnabled: false),
+            TestCase(optedInInParent: false, disabledInParent: false, optedInInChild: true, disabledInChild: false, isEnabled: true),
+            TestCase(optedInInParent: true, disabledInParent: false, optedInInChild: true, disabledInChild: false, isEnabled: true),
+            TestCase(optedInInParent: false, disabledInParent: true, optedInInChild: true, disabledInChild: false, isEnabled: true),
+            TestCase(optedInInParent: true, disabledInParent: true, optedInInChild: true, disabledInChild: false, isEnabled: true),
+            TestCase(optedInInParent: false, disabledInParent: false, optedInInChild: false, disabledInChild: true, isEnabled: false),
+            TestCase(optedInInParent: true, disabledInParent: false, optedInInChild: false, disabledInChild: true, isEnabled: false),
+            TestCase(optedInInParent: false, disabledInParent: true, optedInInChild: false, disabledInChild: true, isEnabled: false),
+            TestCase(optedInInParent: true, disabledInParent: true, optedInInChild: false, disabledInChild: true, isEnabled: false),
+            TestCase(optedInInParent: false, disabledInParent: false, optedInInChild: true, disabledInChild: true, isEnabled: false),
+            TestCase(optedInInParent: true, disabledInParent: false, optedInInChild: true, disabledInChild: true, isEnabled: false),
+            TestCase(optedInInParent: false, disabledInParent: true, optedInInChild: true, disabledInChild: true, isEnabled: false),
+            TestCase(optedInInParent: true, disabledInParent: true, optedInInChild: true, disabledInChild: true, isEnabled: false)
+            // swiftlint:enable line_length
+        ]
+        XCTAssertEqual(testCases.unique.count, 4 * 4)
+        let ruleType = ImplicitReturnRule.self
+        let ruleIdentifier = ruleType.description.identifier
+        for testCase in testCases {
+            let parentConfiguration = Configuration(rulesMode: .default(
+                disabled: testCase.disabledInParent ? [ruleIdentifier] : [],
+                optIn: testCase.optedInInParent ? [ruleIdentifier] : []
+            ))
+            let childConfiguration = Configuration(rulesMode: .default(
+                disabled: testCase.disabledInChild ? [ruleIdentifier] : [],
+                optIn: testCase.optedInInChild ? [ruleIdentifier] : []
+            ))
+            let mergedConfiguration = parentConfiguration.merged(withChild: childConfiguration, rootDirectory: "")
+            let isEnabled = mergedConfiguration.contains(rule: ruleType)
+            let message = "optedInInParent = \(testCase.optedInInParent) " +
+                          "disabledInParent = \(testCase.disabledInParent), " +
+            "optedInInChild = \(testCase.optedInInChild), disabledInChild = \(testCase.disabledInChild)"
```

Comment:
```suggestion
            let message = "optedInInParent = \(testCase.optedInInParent) " +
                          "disabledInParent = \(testCase.disabledInParent), " +
                          "optedInInChild = \(testCase.optedInInChild), " +          
                          "disabledInChild = \(testCase.disabledInChild)"
```

Alternatively, you may add the message to the `TestCase` struct as a computed property which would avoid the `testCase.`.